simulariumio/tests/converters/test_physicell_converter.py

```diff
@@ -206,6 +206,162 @@
                 "plotData": {"version": CURRENT_VERSION.PLOT_DATA, "data": []},
             },
         ),
+        # 3 cells 1 frame
+        (
+            PhysicellData(
+                timestep=360.0,
+                path_to_output_dir="simulariumio/tests/data/physicell/output/",
+                meta_data=MetaData(
+                    box_size=np.array([1000.0, 1000.0, 100.0]),
+                    scale_factor=0.01,
+                ),
+                nth_timestep_to_read=2,
+                display_data={
+                    0: DisplayData(
+                        name="Cancer cell",
+                        radius=30.0,
+                        color="#0080ff",
+                    ),
+                },
+                phase_names={1: {4: "interphase"}},
+            ),
```

Comment:
I think this test is supposed to be testing the skipping frames feature, but there are so much data here, that as a developer, if this test failed, it wouldn't immediately tell me if I broke the skipping frames feature, or if some other default thing just changed and I need to update the data. It would be nice to have a separate function that tests this feature as a unit test without all the other data in the output file. Also it would make this file a lot easier to read. For instance, you don't need to keep checking cameraDefaults

Reply:
out of scope for this PR I think, I should do that for all my tests. Can you submit a maintenance issue?

Reply:
Ok, up to you, but personally I think you could write a couple tests in a clearer format for this new feature, so that the PR includes both new feature + final tests of that new feature. Instead of placeholders that you know you're going to re-write. I'll make an issue

Reply:
https://github.com/simularium/simulariumio/issues/98